2020-09-23 10:55:30,772 ERROR (Thread-18) [__main__] __init__() got an unexpected keyword argument 'ensure_ASCII'
2020-09-23 10:55:39,689 ERROR (Thread-19) [__main__] __init__() got an unexpected keyword argument 'ensure_ASCII'
2020-09-23 10:55:58,549 ERROR (Thread-20) [__main__] __init__() got an unexpected keyword argument 'ensure_ASCII'
2020-09-23 10:59:07,934 ERROR (Thread-176) [__main__] (sqlite3.IntegrityError) FOREIGN KEY constraint failed
[SQL: INSERT INTO id_to_name (point_id, dev_name) VALUES (?, ?)]
[parameters: ('5', '다섯번째입력')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-09-23 13:48:54,972 ERROR (Thread-93) [__main__] (sqlite3.IntegrityError) FOREIGN KEY constraint failed
[SQL: INSERT INTO id_to_name (point_id, dev_name) VALUES (?, ?)]
[parameters: ('==ID 입력==', '==이름 입력==')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-09-23 13:48:57,250 ERROR (Thread-95) [__main__] (sqlite3.IntegrityError) FOREIGN KEY constraint failed
[SQL: INSERT INTO id_to_name (point_id, dev_name) VALUES (?, ?)]
[parameters: ('==ID 입력==', '==이름 입력==')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-09-23 13:48:58,795 ERROR (Thread-97) [__main__] (sqlite3.IntegrityError) FOREIGN KEY constraint failed
[SQL: INSERT INTO id_to_name (point_id, dev_name) VALUES (?, ?)]
[parameters: ('==ID 입력==', '==이름 입력==')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
